test/NuGet.Core.FuncTests/NuGet.Packaging.FuncTest/SigningTests/SignatureTests.cs

```diff
@@ -142,7 +138,7 @@ public async Task GetSigningCertificateFingerprint_SuccessfullyHashesMultipleAlg
         }
 
         [Fact]
-        public async Task Timestamps_WithTwoAttributesAndOneValueEach_ReturnsTwoTimestamps()
```

Comment:
I removed this test because without BouncyCastle, it became impractical to test and offered little value.

[RFC 5652](https://www.rfc-editor.org/rfc/rfc5652#section-5.3) defines this ASN.1:

```text
      UnsignedAttributes ::= SET SIZE (1..MAX) OF Attribute

      Attribute ::= SEQUENCE {
        attrType OBJECT IDENTIFIER,
        attrValues SET OF AttributeValue }

      AttributeValue ::= ANY
```

An attribute has an object identifier (OID) that identifies its purpose, like [`1.2.840.113549.1.9.6` for countersignature](https://www.rfc-editor.org/rfc/rfc5652#section-11.4).  It is possible to have multiple attribute values for the same OID via:

* two separate attributes with the same OID, each attribute with its own value
* one attribute with multiple values

Sometimes RFC's permit multiple values, like RFC 5652 does with the countersignature attribute:

   > A countersignature attribute can have multiple attribute values.  The
   syntax is defined as a SET OF AttributeValue, and there MUST be one
   or more instances of AttributeValue present.
  >
  >  The UnsignedAttributes syntax is defined as a SET OF Attributes.  The
   UnsignedAttributes in a signerInfo may include multiple instances of
   the countersignature attribute.

Other times, RFC's may explicitly prohibit multiple values.  Example from [RFC 5652](https://www.rfc-editor.org/rfc/rfc5652#section-11.3):

   > A signing-time attribute MUST have a single attribute value, even
   though the syntax is defined as a SET OF AttributeValue.  There MUST
   NOT be zero or multiple instances of AttributeValue present.

It is easy to create all of these possibilities with System.Formats.Asn1.  However, when loading the ASN.1 into a `SignedCms` class, the `SignedCms` implementation uses Windows crypto API's to decode the ASN.1, and Windows has a tendency to normalize attributes and eliminate unnecessary duplication.

This test was one of a few that I removed because it became unnecessarily complicated to port the test and circumvent Windows normalizing behavior.